acc_test/resource_cluster_test.go

```diff
@@ -363,6 +363,51 @@ func TestAccOpenSearchClusterResize(t *testing.T) {
 	})
 }
 
+func TestAccRedisClusterResize(t *testing.T) {
```

Comment:
The acc tests are slow and flakey at the moment, largely because there are a bunch of tests that wait until a cluster reaches running, which takes a long time.
Could you please make sure that you aren't creating more REDIS clusters than is absolutely necessary. i.e. If there are more than one test cases that make REDIS clusters and wait for them to reach running, consider consolidating them (Yu has done this for Kafka in the other open PR at the moment).

Reply:
I have combined Redis tests together in "resource_redis_test.go"(in new commit).
Both "cluster create" and "resize" tests requires running status, they are merged into one case to save time.